### PR TITLE
Fix some doc nits

### PR DESCRIPTION
**Pull request by richsalz**

### CODE REVIEW

File: doc/man3/UI_STRING.pod [hunk 1]
```diff
@@ -1,8 +1,13 @@
...
-
-UI_STRING, UI_string_types,
+=head1 NAME
+
+UI_get_input_flags, UI_get0_output_string,
+UI_get0_action_string, UI_get0_result_string,
+UI_get0_test_string, UI_get_result_minsize,
+UI_get_result_maxsize, UI_set_result,
+UI_STRING, UI_string_types, UI_get_string_type,
+UI_STRING, UI_string_types - User interface string parsing
```
Comment (levitte) [inline doc/man3/UI_STRING.pod]: Some names doubled, and if I may, I would prefer to have `UI_STRING` first.

**Comment by richsalz** [comment]
doubling removed updated commit pushed.

**Comment by levitte** [comment]
Thank you for picking this up :-)

**Comment by richsalz** [comment]
master d4ea965 and 1.1.0 0d8c225.

**Comment by levitte** [comment]
Don't forget to push 1.1.0 too ;-)

**Pull request closed without merging**